From cd5bc7da855fd27b4c74e6c216b95fc5444123f2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Niklas=20Ekstr=C3=B6m?= <mail@niklasekstrom.nu>
Date: Wed, 29 May 2024 09:35:19 +0200
Subject: [PATCH 06/13] Set library version to 3.0

The version for bsdsocket.library must be at least 3 to work.

diff --git a/Software/libremote/library.c b/Software/libremote/library.c
index bd30bf5..b4777d5 100644
--- a/Software/libremote/library.c
+++ b/Software/libremote/library.c
@@ -24,7 +24,7 @@
 #define BOUNCE_BUFFER_SIZE  (4*1024)
 
 // Must keep VERSION in romtag.asm in sync.
-#define LIB_VERSION     1
+#define LIB_VERSION     3
 #define LIB_REVISION    0
 #define LIB_DATE        "28.5.2024"
 
@@ -32,7 +32,7 @@
 #define STR(s) #s
 
 const char library_name[] = LIBRARY_NAME;
-const char id_string[] = LIBRARY_NAME " " VSTR(LIB_VERSION) "." VSTR(LIB_REVISION) " (" LIB_DATE ")";
+const char id_string[] = LIBRARY_NAME " " VSTR(LIB_VERSION) "." VSTR(LIB_REVISION) " (" LIB_DATE ")\n\r";
 const char service_name[] = SERVICE_NAME;
 
 static BPTR saved_seg_list;
diff --git a/Software/libremote/romtag.asm b/Software/libremote/romtag.asm
index c2f1d78..8e8ec74 100644
--- a/Software/libremote/romtag.asm
+++ b/Software/libremote/romtag.asm
@@ -5,7 +5,7 @@
 RTC_MATCHWORD:  equ     $4afc
 RTF_AUTOINIT:   equ     (1<<7)
 NT_LIBRARY:     equ     9
-VERSION:        equ     1
+VERSION:        equ     3
 PRIORITY:       equ     0
 
     section CODE,CODE
-- 
2.47.3

